cli/test/README.md

```diff
@@ -423,21 +439,33 @@ This document includes CLI commands and the expected CLI output in order to help
+|`fastddsqosprof file.xml set participant[profile].use_builtin_transports other`|`Fast DDS QoS Profiles Manager exception caught: value 'other' is invalid boolean`|N/A|
 |`fastddsqosprof file.xml set participant[profile].user_transports[]`|`ERROR: set command for Participant <user_transports> expects 1 arguments and received 0`|`PARTICIPANT_USER_TRANSPORTS_USAGE`|
 |`fastddsqosprof file.xml set participant[profile].user_transports[].subelement`|`ERROR: Participant <user_transports> must be FINAL element`|`PARTICIPANT_USER_TRANSPORTS_USAGE`|
 |`fastddsqosprof file.xml set participant[profile].user_transports[] -h`|N/A|`PARTICIPANT_USER_TRANSPORTS_USAGE`|
 |`fastddsqosprof file.xml set participant[profile].user_transports[] --help`|N/A|`PARTICIPANT_USER_TRANSPORTS_USAGE`|
 |`fastddsqosprof file.xml set participant[profile].user_transports[] help`|N/A|`PARTICIPANT_USER_TRANSPORTS_USAGE`|
 |`fastddsqosprof file.xml set participant[profile].user_transports[] arg1 arg2 -h`|N/A|`PARTICIPANT_USER_TRANSPORTS_USAGE`|
 |`fastddsqosprof file.xml set participant[profile].user_transports[] arg1 arg2`|`ERROR: set command for Participant <user_transports> expects 1 arguments and received 2`|`PARTICIPANT_USER_TRANSPORTS_USAGE`|
-|`fastddsqosprof file.xml set participant[profile].user_transports[] argument`|**UNSUPPORTED** (currently `Fast DDS QoS Profiles Manager exception caught: Unsupported`)|N/A|
-|`fastddsqosprof file.xml set participant[profile].user_transports[11811] argument`|**UNSUPPORTED** (currently `Fast DDS QoS Profiles Manager exception caught: Unsupported`)|N/A|
+|`fastddsqosprof file.xml set participant[profile].user_transports[] arg1`|N/A|N/A|
+|`fastddsqosprof file.xml set participant[profile].user_transports[0] arg1`|N/A|N/A|
+|`fastddsqosprof file.xml set participant[profile].user_transports[11811] argument`|`Fast DDS QoS Profiles Manager exception caught: userTransports does not have an element in position 11811`|N/A|
 
 [^1]: **PENDING**: Print and clear default profile command will not require a profile name.
 [^2]: **PENDING**: Print, clear and query commands should not allow an empty index with a subelement.
+[^3]: **PENDING**: Access to file before previous changes have been written although Xerces has correctly return from `write` operation.
 
 ## Caveats
 
 This current test plan is focused only on the CLI usage and errors.
 Currently, this test plan is not concerned about the validity of the arguments passed to Fast DDS QoS Profiles Manager Library, nor about the handling of the returned information.
+
+
+## Known issues
+
+* Simultaneous `set` operations would lead to an unexpected exception reporting the following message: `Fast DDS QoS Profiles Manager exception caught: invalid document structure`.
+  This happens due to accessing to file before previous changes have been written although Xerces has correctly return from `write` operation.
+  This issue has been marked as pending with operand [^3].
```

Comment:
I do not think this is needed and it can be confusing. I think we should remove this sentence.